include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_for.h

```diff
@@ -181,6 +179,9 @@ __future<sycl::event>
 __parallel_for(oneapi::dpl::__internal::__device_backend_tag, _ExecutionPolicy&& __exec, _Fp __brick, _Index __count,
                _Ranges&&... __rngs)
 {
+    assert(std::min({std::make_unsigned_t<std::common_type_t<oneapi::dpl::__internal::__difference_t<_Ranges>...>>(__rngs.size())...}) > 0);
```

Comment:
optional nitpick:  
In other places you have created an alias for the size type in this situation and it is significantly easier to read, perhaps we should do that here.

Reply:
Yes, before was an alias... But we have got a warning about unused alias, because `assert` is changed by nothing in release configuration, for example. 
So, I removed this alias to avoid the warning.